src/index.html

```diff
@@ -20,8 +21,437 @@
       href="styles/index.scss"
     />
   </head>
+  <!-- #endregion -->
+  <body class="page">
+    <!-- #region header -->
+    <header class="header">
+      <img
+        class="header-logo"
+        src="images/logo.png"
+        alt="logo"
+      />
 
-  <body>
-    <h1>Catalog</h1>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__list-item">
+            <a
+              href="#apple"
+              class="is-active nav__link"
+            >
+              APPLE
+            </a>
+          </li>
+          <li class="nav__list-item">
+            <a
+              href="#samsung"
+              class="nav__link"
+            >
+              SAMSUNG
+            </a>
+          </li>
+          <li class="nav__list-item">
+            <a
+              href="#smartphones"
+              class="nav__link"
+            >
+              SMARTPHONES
+            </a>
+          </li>
+          <li class="nav__list-item">
+            <a
+              class="nav__link"
+              href="#laptops & computers"
```

Comment:
The `href` attribute of the anchor tag contains an invalid value. The space character is not allowed in URLs, and `&` should be encoded as `&amp;`.